src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -506,6 +510,36 @@ public void delete() throws IOException {
     }
 
     /**
+     * Sort orders for listing forks
+     */
+    public static enum Sort { NEWEST, OLDEST, STARGAZERS }
```

Comment:
You shouldn't copypaste all code and change only request, you should implement new feature in the most actual way. Or this lib became unmaintainable soon.